app/consumer/genesis.go

```diff
@@ -33,24 +33,16 @@ type GenesisState map[string]json.RawMessage
 type IcsVersion string
 
 const (
-	v2_x   IcsVersion = "v2.x"
-	v3_0_x IcsVersion = "v3.0.x"
-	v3_1_x IcsVersion = "v3.1.x"
-	v3_2_x IcsVersion = "v3.2.x"
-	v3_3_x IcsVersion = "v3.3.x"
 	v4_x_x IcsVersion = "v4.x"
+	v5_x_x IcsVersion = "v5.x"
 )
 
 var TransformationVersions map[string]IcsVersion = map[string]IcsVersion{
-	"v2.x":   v2_x,
-	"v3.0.x": v3_0_x,
-	"v3.1.x": v3_1_x,
-	"v3.2.x": v3_2_x,
-	"v3.3.x": v3_3_x,
-	"v4.x":   v4_x_x,
+	"v4.x": v4_x_x,
+	"v5.x": v5_x_x,
 }
 
-// Transformation of consumer genesis content as it is exported from a provider version v1,2,3
+// Transformation of consumer genesis content as it is exported from a provider version v4
 // to a format readable by current consumer implementation.
```

Comment:
What's the current consumer implementation? Could you please specify the consumer version? Is this transformation still needed?